src/index.html

```diff
@@ -7,8 +7,51 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img 
+          src="images/logo.png"
+          alt="img"
+          class="header__logo"
+        >
+      </a>
+      <nav class="nav">
```

Comment:
add an empty line between _multiline_ siblings

Reply:
https://github.com/mate-academy/layout_moyo-header/pull/3081/files#r1184155835
not fixed